### DISCUSSION TITLE
Google login: Implement in registration flow

### DISCUSSION THREAD

**Discussion by LiveNathan** [wimdeblauwe/wimdeblauwe.com#6] [Ideas]
Hi Wim, thanks for this article. I'd love it if you would expand this into the a complete user registration/login flow, maybe using the example from your book. I have tried to implement this twice using other tutorials online and have not been able to figure it out. Getting the initial OAuth stuff set up is easy. It's the connection with the user database afterwards that I don't understand how to implement.

**Comment by wimdeblauwe** [comment]
The basic idea is that you will normally get some id from the social login provider and you store that in your own user table to couple the social login with the user in your database.
